nixos/modules/rename.nix

```diff
@@ -284,6 +284,12 @@ with lib;
     (mkRemovedOptionModule [ "services" "redis" "appendOnlyFilename" ] "This option was never used.")
     (mkRemovedOptionModule [ "services" "redis" "pidFile" ] "This option was removed.")
 
+    # Nvidia
+    (mkRemovedOptionModule [ "hardware" "nvidia" "optimus_prime" "enable" ] [ "hardware" "nvidia" "prime" "sync" "enable" ])
+    (mkRemovedOptionModule [ "hardware" "nvidia" "optimus_prime" "allowExternalGpu" ] [ "hardware" "nvidia" "prime" "sync" "allowExternalGpu" ])
+    (mkRemovedOptionModule [ "hardware" "nvidia" "optimus_prime" "nvidiaBusId" ] [ "hardware" "nvidia" "prime" "nvidiaBusId" ])
+    (mkRemovedOptionModule [ "hardware" "nvidia" "optimus_prime" "intelBusId" ] [ "hardware" "nvidia" "prime" "intelBusId" ])
```

Comment:
These should be `mkRenamedOptionModule`, otherwise there will be an errors
`error: cannot coerce a list to a string`
for those using old options (as `mkRemovedOptionModule` expect string instead of list as its second argument)